modules/media/jsx/CandidateMediaWidget.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * React component for a widget on the candidate dashboard
+ * displaying the media associated with the candidate.
+ *
+ * @param {array} props - The React props
+ *
+ * @return {object} - The component
+ */
+function CandidateMediaWidget(props) {
+    let files = [];
+    for (let i = 0; i < props.Files.length; i++) {
+        const file = props.Files[i];
+        files.push(
+            <a className="list-group-item" key={i}
+                href={props.BaseURL
+                    + '/media/ajax/FileDownload.php?File='
+                    + encodeURIComponent(file.Filename)}>
```

Comment:
you should get the config media path and append it here

Reply:
otherwise i am getting 404 error

Reply:
Are you sure? This is the same way it's done in mediaIndex.js:

```
        const downloadURL = loris.BaseURL + '/media/ajax/FileDownload.php?File=' +
          encodeURIComponent(row['File Name']);
```

Reply:
you're right. might have something to do with how i upgraded. ill investigate and let you know if it's actually an issue with the PR